src/jit/CMakeLists.txt

```diff
@@ -6,11 +6,12 @@ include_directories("../inc")
 # Enable the following for UNIX altjit on Windows
 # add_definitions(-DALT_JIT)
 
-if (WIN32) 
 if (CLR_CMAKE_PLATFORM_ARCH_AMD64)
-  add_definitions(-DFEATURE_SIMD -DFEATURE_AVX_SUPPORT) 
-endif (CLR_CMAKE_PLATFORM_ARCH_AMD64)
+  add_definitions(-DFEATURE_SIMD) 
```

Comment:
How are we validating that all SIMD intrinsics are getting honored and there is a perf benefit with SIMD turned ON?  Are we manually checking the code generated for some of SIMD unit tests? Sometime back we had a work item to have a debug mode complus setting to assert that JIT never gets to compile a method which is considered SIMD intrinsic.  We should open a Github issue for tracking it.